source/plugins/ruby/filter_cadvisor2mdm.rb

```diff
@@ -161,19 +163,30 @@ def filter(tag, time, record)
             if counter_name == Constants::CPU_USAGE_NANO_CORES
               metric_name = Constants::CPU_USAGE_MILLI_CORES
               metric_value /= 1000000 #cadvisor record is in nanocores. Convert to mc
-              @log.info "Metric_value: #{metric_value} CPU Capacity #{@cpu_capacity}"
-              if @cpu_capacity != 0.0
-                percentage_metric_value = (metric_value) * 100 / @cpu_capacity
+              target_node_cpu_capacity_nc = @NodeCache.cpu.get_capacity(record["DataItems"][0]["Host"])
```

Comment:
we are invoking getNodesResourceUri in ensure_cpu_memory_capacity_set when controller_type.downcase == "replicaset". is this additional call not required any more since we will be using from cache?